workspaces/rbac/plugins/rbac-backend/src/service/permission-definition-routes.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  PermissionDependentPluginList,
+  policyEntityCreatePermission,
+  policyEntityDeletePermission,
+  policyEntityReadPermission,
+} from '@backstage-community/plugin-rbac-common';
+import Router from 'express-promise-router';
+import { logAuditorEvent, setAuditorError } from '../auditor/rest-interceptor';
+import { PluginPermissionMetadataCollector } from './plugin-endpoints';
+import {
+  PermissionDependentPluginDTO,
+  PermissionDependentPluginStore,
+} from '../database/extra-permission-enabled-plugins-storage';
+import { authorizeConditional } from './policies-rest-api';
+import {
+  AuditorService,
+  AuthService,
+  HttpAuthService,
+  PermissionsService,
+} from '@backstage/backend-plugin-api';
+import { ExtendablePluginIdProvider } from './extendable-id-provider';
+import {
+  ConflictError,
+  NotAllowedError,
+  NotFoundError,
+} from '@backstage/errors';
+
+// todo: do we need separated permissions to set up extra plugin ids  ?
```

Comment:
I don't think we do at the moment. This still be a part of our current permissions and we can just wait for a request to add additional permissions for these individual tasks whenever we get requests.